lib/archethic/contracts/interpreter/condition_interpreter.ex

```diff
@@ -18,9 +18,14 @@ defmodule Archethic.Contracts.Interpreter.ConditionInterpreter do
   @doc """
   Parse the given node and return the trigger and the actions block.
   """
-  @spec parse(any()) ::
+  @spec parse(any(), list) ::
           {:ok, condition_type(), Conditions.t()} | {:error, any(), String.t()}
-  def parse(node = {{:atom, "condition"}, _, [[{{:atom, condition_name}, keyword}]]}) do
+  def parse(_, _ \\ [])
```

Comment:
I think this default value is not necessary since the function_key are always passed as parameter